Unable to locate executable file: 'mkdir' on windows host

I am trying to use the extension from dev.azure.com on an on-premise windows 2012 R2 host.
The log contains:
2019-09-06T15:47:39.6476413Z ##[section]Starting: oc-cmd 
2019-09-06T15:47:39.7498004Z ==============================================================================
2019-09-06T15:47:39.7498316Z Task         : Execute oc command
2019-09-06T15:47:39.7498464Z Description  : OpenShift execute CLI command.
2019-09-06T15:47:39.7498592Z Version      : 2.0.12
2019-09-06T15:47:39.7498748Z Author       : Hardy Ferentschik
2019-09-06T15:47:39.7498931Z Help         : See [getting started](https://github.com/redhat-developer/openshift-vsts/blob/master/docs/getting-started.md)
2019-09-06T15:47:39.7499106Z ==============================================================================
2019-09-06T15:47:40.6323174Z ##[debug]agent.TempDirectory=C:\DevOpsAgent\_work\_temp
2019-09-06T15:47:40.6484718Z ##[debug]loading inputs and endpoints
2019-09-06T15:47:40.6520288Z ##[debug]loading ENDPOINT_AUTH_5d46810a-dcb1-4ca0-bb4e-170cf29b22d8
2019-09-06T15:47:40.6553258Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_5d46810a-dcb1-4ca0-bb4e-170cf29b22d8_ACCEPTUNTRUSTEDCERTS
2019-09-06T15:47:40.6555368Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_5d46810a-dcb1-4ca0-bb4e-170cf29b22d8_APITOKEN
2019-09-06T15:47:40.6555882Z ##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
2019-09-06T15:47:40.6556272Z ##[debug]loading ENDPOINT_AUTH_SCHEME_5d46810a-dcb1-4ca0-bb4e-170cf29b22d8
2019-09-06T15:47:40.6556614Z ##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
2019-09-06T15:47:40.6557812Z ##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
2019-09-06T15:47:40.6558008Z ##[debug]loading INPUT_CMD
2019-09-06T15:47:40.6558217Z ##[debug]loading INPUT_OPENSHIFTSERVICE
2019-09-06T15:47:40.6558414Z ##[debug]loading INPUT_VERSION
2019-09-06T15:47:40.6558615Z ##[debug]loaded 10
2019-09-06T15:47:40.6558819Z ##[debug]Agent.ProxyUrl=undefined
2019-09-06T15:47:40.6558996Z ##[debug]Agent.CAInfo=undefined
2019-09-06T15:47:40.6559195Z ##[debug]Agent.ClientCert=undefined
2019-09-06T15:47:40.6559403Z ##[debug]Agent.SkipCertValidation=undefined
2019-09-06T15:47:40.6559602Z ##[debug]version=v3.9.0
2019-09-06T15:47:40.6559905Z ##[debug]cmd=start-build s2i-art-portal -n art-portal
2019-09-06T15:47:40.6560117Z ##[debug]creating download directory
2019-09-06T15:47:40.6560299Z ##[debug]which 'mkdir'
2019-09-06T15:47:40.6560491Z ##[debug]not found
2019-09-06T15:47:40.6560686Z ##[debug]task result: Failed
2019-09-06T15:47:40.6612305Z ##[error]Unable to locate executable file: 'mkdir'. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also verify the file has a valid extension for an executable file.
2019-09-06T15:47:40.6622072Z ##[debug]Processed: ##vso[task.issue type=error;]Unable to locate executable file: 'mkdir'. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also verify the file has a valid extension for an executable file.
2019-09-06T15:47:40.6642399Z ##[debug]Processed: ##vso[task.complete result=Failed;]Unable to locate executable file: 'mkdir'. Please verify either the file path exists or the file can be found within a directory specified by the PATH environment variable. Also verify the file has a valid extension for an executable file.
2019-09-06T15:47:40.6672076Z ##[section]Finishing: oc-cmd 

Kindly
Gianluca

--- a/src/oc-install.ts
+++ b/src/oc-install.ts
@@ -39,9 +39,7 @@ export async function installOc(
   let downloadDir =
     process.env['SYSTEM_DEFAULTWORKINGDIRECTORY'] + '/.download';
   if (!fs.existsSync(downloadDir)) {
-    let mkdir: ToolRunner = tl.tool('mkdir');
-    mkdir.arg('-p').arg(downloadDir);
-    await mkdir.exec();
+    tl.mkdirP(downloadDir);
   }
 
   let url: string | null;

